Source/JavaScriptCore/b3/air/AirLowerAfterRegAlloc.cpp

```diff
@@ -113,7 +113,7 @@ void lowerAfterRegAlloc(Code& code)
     // kind of slop is OK.
     RegisterSet disallowedCalleeSaves;
     if (code.stackIsAllocated()) {
-        disallowedCalleeSaves = RegisterSet::calleeSaveRegisters();
+        disallowedCalleeSaves = RegisterSet::vmCalleeSaveRegisters();
```

Comment:
Why is this change needed?

Reply:
To match the corresponding change in `AirCode`--I'm pretty sure geza made this change to fix a bug, but it seems to me that the jit ABI is the correct one to be using here, at least, as far as I understand it, the native ABI isn't the relevant one.

FWIW the other architectures with Air support don't have as much skew between the two ABIs as armv7 does,  (if i am reading right, both x86_64 on non-windows and aarch64 have nearly identical `{vmCallee,callee}saveRegister` sets, only the frame pointer is different among the GPRs) ...so it makes sense we'd be surfacing this bug now.